packages/front-end/components/SyntaxHighlighting/Snippets/TargetingAttributeCodeSnippet.tsx

```diff
@@ -62,8 +62,7 @@ function getExampleAttributes({
         ? ["foo", "bar"].map((v) => sha256(v, secureAttributeSalt))
         : ["foo", "bar"];
     } else if (datatype === "enum") {
-      // @ts-expect-error TS(2532) If you come across this, please fix it!: Object is possibly 'undefined'.
-      value = enumList.split(",").map((v) => v.trim())[0] ?? null;
+      value = enumList?.split(",").map((v) => v.trim())[0] ?? null;
```

Comment:
Tested in `node -i` that this still behaves appropriately when enumList is either `string` or `undefined`:

```javascript
> "foo"?.split(",").map(v => v.trim())[0] ?? null
'foo'
> undefined?.split(",").map(v => v.trim())[0] ?? null
null
```